lib/cached_counts.rb

```diff
@@ -147,6 +155,13 @@ def define_scope_count_attribute(attribute_name, relation, options)
       end
     end
 
+    def default_race_condition_fallback_proc(key, relation, options)
+      fallback = Rails.cache.read(key)
+      fallback = fallback.value if fallback.is_a?(ActiveSupport::Cache::Entry)
```

Comment:
Looks like this gives nil when no data is present at the key, so this should be good.

If you wanted to be extra careful you could explicitly assign nil in the else case